src/components/video-editor/VideoEditor.tsx

```
@@ -376,25 +386,33 @@ export default function VideoEditor() {
 	const [audioRegions, setAudioRegions] = useState<AudioRegion[]>([]);
 	const [selectedAudioId, setSelectedAudioId] = useState<string | null>(null);
 	const [autoCaptions, setAutoCaptions] = useState<CaptionCue[]>([]);
-	const [autoCaptionSettings, setAutoCaptionSettings] = useState<AutoCaptionSettings>(
-		DEFAULT_AUTO_CAPTION_SETTINGS,
-	);
+	const [selectedCaptionId, setSelectedCaptionId] = useState<string | null>(null);
+	const [timeSelection, setTimeSelection] = useState<TimeSelection | null>(null);
+	const [autoCaptionSettings, setAutoCaptionSettings] = useState<AutoCaptionSettings>(() => ({
+		...DEFAULT_AUTO_CAPTION_SETTINGS,
+		selectedModel: (initialEditorPreferences.whisperSelectedModel as any) || "small",
```

Biome (2.4.9)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)